cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/users/benda12/Emulated_NDN_Testbed_in_ONL/configuration/nfdDataCollection/.waf-1.8.0-094bcb3e7079041614f60222067e94d3/waflib/Tools/c_config.py", line 478, in run_c_code\n    bld.compile()\n  File "/users/benda12/Emulated_NDN_Testbed_in_ONL/configuration/nfdDataCollection/.waf-1.8.0-094bcb3e7079041614f60222067e94d3/waflib/Build.py", line 182, in compile\n    raise Errors.BuildError(self.producer.error)\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 46410576: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-Werror\', \'-std=c++11\', \'../test.cpp\', \'-c\', \'-o\', \'/users/benda12/Emulated_NDN_Testbed_in_ONL/configuration/nfdDataCollection/build/.conf_check_b150c0600ccd48f9dd5ae13ba2d58321/testbuild/test.cpp.1.o\']\n'
